src/BigQueryBundle/BigQuery/UnitOfWork.php

```diff
@@ -7,6 +7,7 @@
 use CCMBenchmark\BigQueryBundle\BigQuery\Entity\RowInterface;
 use CCMBenchmark\BigQueryBundle\CloudStorage\FileSystem;
 use CCMBenchmark\BigQueryBundle\CloudStorage\FileSystemInterface;
+use Google_Service_Bigquery_QueryRequest;
```

Comment:
The clients now offers a proper namespace, can you rely on `Google\Service\BigQuery\QueryRequest` (double check this name) instead ?